routes/@checkout.js

```diff
@@ -709,6 +721,18 @@ const schema = {
             },
             'description': 'Gift wrap chosen by customer'
           },
+          'currency_id': {
+            'type': 'string',
+            'pattern': '^[A-Z]{3}$',
+            'default': 'BRL',
+            'description': 'Currency ID'
+          },
+          'currency_symbol': {
+            'type': 'string',
+            'maxLength': 20,
+            'default': 'R$',
+            'description': 'Currency Symbol'
+          },
```

Comment:
Se for colocar dentro da `transaction` apenas (em vez do root do corpo do checkout), deveria adicionar no objeto `item` separadamente também